build-system/babel-config/unminified-config.js

```diff
@@ -38,7 +38,10 @@ function getUnminifiedConfig() {
   const replacePlugin = getReplacePlugin();
   const unminifiedPlugins = [
     './build-system/babel-plugins/babel-plugin-jsx-style-object',
-    getImportResolverPlugin(),
+    getImportResolverPlugin({
```

Comment:
This is the closest reproduction I can find of the error I get when I add these aliases to the configuration. It seems likely that the build system is passing a different version of preact to the one we are using in the codebase. https://github.com/preactjs/preact/issues/3220